<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="32" passed="11" failed="21" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-19T17:06:15 PST" name="RegressionSuite" finished-at="2024-02-19T17:12:35 PST" duration-ms="379904">
    <groups>
    </groups>
    <test started-at="2024-02-19T17:06:15 PST" name="TestLogin" finished-at="2024-02-19T17:07:03 PST" duration-ms="47763">
      <class name="com.salesforce.automation.SalesforceLoginPage_Testcases">
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:15 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:06:20 PST" duration-ms="4260" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="loginPage_EmptyPassword()[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:20 PST" name="loginPage_EmptyPassword" finished-at="2024-02-19T17:06:21 PST" duration-ms="958" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPage_EmptyPassword -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:21 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:06:21 PST" duration-ms="842" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:21 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:06:25 PST" duration-ms="3251" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="testForgotPasssword()[pri:1, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:25 PST" name="testForgotPasssword" finished-at="2024-02-19T17:06:26 PST" duration-ms="1102" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPasssword -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:26 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:06:26 PST" duration-ms="637" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:26 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:06:30 PST" duration-ms="3737" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="loginErrorMessage()[pri:2, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:30 PST" name="loginErrorMessage" finished-at="2024-02-19T17:06:46 PST" duration-ms="16011" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: div#error.loginError
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 120.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20231129155202, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 20.5.0, moz:processID: 49009, moz:profile: /var/folders/gx/qjsbcn6d6d3..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: e7da0151-6709-46b7-9355-aa26aa237928
*** Element info: {Using=css selector, value=div#error.loginError}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: div#error.loginError
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 120.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20231129155202, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 20.5.0, moz:processID: 49009, moz:profile: /var/folders/gx/qjsbcn6d6d3..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: e7da0151-6709-46b7-9355-aa26aa237928
*** Element info: {Using=css selector, value=div#error.loginError}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:420)
at org.openqa.selenium.By$ByCssSelector.findElement(By.java:431)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.salesforce.automation.SalesforceLoginPage_Testcases.loginErrorMessage(SalesforceLoginPage_Testcases.java:142)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorMessage -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:46 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:06:47 PST" duration-ms="695" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:47 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:06:51 PST" duration-ms="3649" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="login_Homepage()[pri:3, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:51 PST" name="login_Homepage" finished-at="2024-02-19T17:06:55 PST" duration-ms="4198" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceLoginPage_Testcases.login_Homepage(SalesforceLoginPage_Testcases.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_Homepage -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:55 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:06:56 PST" duration-ms="1009" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:06:56 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:07:00 PST" duration-ms="3752" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="rememberUser_checkbox()[pri:4, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:07:00 PST" name="rememberUser_checkbox" finished-at="2024-02-19T17:07:02 PST" duration-ms="1978" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found [Login | Salesforce]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found [Login | Salesforce]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceLoginPage_Testcases.rememberUser_checkbox(SalesforceLoginPage_Testcases.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rememberUser_checkbox -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLoginPage_Testcases@38831718]" started-at="2024-02-19T17:07:02 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:07:03 PST" duration-ms="982" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
      </class> <!-- com.salesforce.automation.SalesforceLoginPage_Testcases -->
    </test> <!-- TestLogin -->
    <test started-at="2024-02-19T17:07:03 PST" name="TestOppurtunities" finished-at="2024-02-19T17:08:25 PST" duration-ms="81791">
      <class name="com.salesforce.automation.SalesforceCreateOpty_Testcases">
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:03 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:07:05 PST" duration-ms="2229" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="OpportunityPipelineReport()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:05 PST" name="OpportunityPipelineReport" finished-at="2024-02-19T17:07:16 PST" duration-ms="11026" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpportunityPipelineReport -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:16 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:07:17 PST" duration-ms="130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:17 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:07:19 PST" duration-ms="2050" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="QuarterlySummaryReport()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:19 PST" name="QuarterlySummaryReport" finished-at="2024-02-19T17:07:28 PST" duration-ms="9167" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at com.salesforce.automation.SalesforceCreateOpty_Testcases.QuarterlySummaryReport(SalesforceCreateOpty_Testcases.java:192)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuarterlySummaryReport -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:28 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:07:28 PST" duration-ms="123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:28 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:07:30 PST" duration-ms="1845" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="StuckOpportunitiesReport()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:30 PST" name="StuckOpportunitiesReport" finished-at="2024-02-19T17:07:38 PST" duration-ms="8198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StuckOpportunitiesReport -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:38 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:07:38 PST" duration-ms="143" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:38 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:07:40 PST" duration-ms="1903" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="createNewOppty()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:40 PST" name="createNewOppty" finished-at="2024-02-19T17:07:55 PST" duration-ms="14979" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://dan000000cz3teag-dev-ed.develop.my.salesforce.com/006an000000lqXp] but found [https://dan000000cz3teag-dev-ed.develop.my.salesforce.com/006/e]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://dan000000cz3teag-dev-ed.develop.my.salesforce.com/006an000000lqXp] but found [https://dan000000cz3teag-dev-ed.develop.my.salesforce.com/006/e]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertCurrentURL(BaseClass.java:273)
at com.salesforce.automation.SalesforceCreateOpty_Testcases.createNewOppty(SalesforceCreateOpty_Testcases.java:102)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewOppty -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:56 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:07:56 PST" duration-ms="59" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:56 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:07:58 PST" duration-ms="1779" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="opportunitiesdropdown()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:07:58 PST" name="opportunitiesdropdown" finished-at="2024-02-19T17:08:05 PST" duration-ms="7413" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- opportunitiesdropdown -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceCreateOpty_Testcases@7203c7ff]" started-at="2024-02-19T17:08:05 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:08:25 PST" duration-ms="20026" status="FAIL">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: failed to close window in 20 seconds
  (Session info: chrome=121.0.6167.184)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57692}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: e126087b22b47ab7c9e8ebc092835e2b]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: failed to close window in 20 seconds
  (Session info: chrome=121.0.6167.184)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57692}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: e126087b22b47ab7c9e8ebc092835e2b
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)
at org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:442)
at com.salesforce.base.BaseClass.driverClose(BaseClass.java:241)
at com.salesforce.base.BaseSalesForceClass.tearDownAfterTestMethod(BaseSalesForceClass.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:823)
at org.testng.internal.invokers.TestInvoker.runAfterConfigurations(TestInvoker.java:792)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:768)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
      </class> <!-- com.salesforce.automation.SalesforceCreateOpty_Testcases -->
    </test> <!-- TestOppurtunities -->
    <test started-at="2024-02-19T17:08:25 PST" name="TestUserMenu" finished-at="2024-02-19T17:09:07 PST" duration-ms="42256">
      <class name="com.salesforce.automation.SalesforceUserMenu_Testcases">
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:25 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:08:29 PST" duration-ms="3518" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="MyProfile()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:29 PST" name="MyProfile" finished-at="2024-02-19T17:08:32 PST" duration-ms="3869" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceUserMenu_Testcases.MyProfile(SalesforceUserMenu_Testcases.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MyProfile -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:33 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:08:33 PST" duration-ms="744" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:33 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:08:36 PST" duration-ms="3105" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="developerConsole()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:36 PST" name="developerConsole" finished-at="2024-02-19T17:08:40 PST" duration-ms="4110" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceUserMenu_Testcases.developerConsole(SalesforceUserMenu_Testcases.java:584)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- developerConsole -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:41 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:08:42 PST" duration-ms="924" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:42 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:08:46 PST" duration-ms="4668" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="logoutOption()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:46 PST" name="logoutOption" finished-at="2024-02-19T17:08:50 PST" duration-ms="3592" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceUserMenu_Testcases.logoutOption(SalesforceUserMenu_Testcases.java:640)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutOption -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:50 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:08:51 PST" duration-ms="896" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:51 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:08:55 PST" duration-ms="3897" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="mySettingOption()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:55 PST" name="mySettingOption" finished-at="2024-02-19T17:08:57 PST" duration-ms="1790" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found [Login | Salesforce]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found [Login | Salesforce]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceUserMenu_Testcases.mySettingOption(SalesforceUserMenu_Testcases.java:341)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mySettingOption -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:57 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:08:58 PST" duration-ms="1113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:08:58 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:09:02 PST" duration-ms="3709" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifySelectuserMenu()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:09:02 PST" name="verifySelectuserMenu" finished-at="2024-02-19T17:09:06 PST" duration-ms="4429" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceUserMenu_Testcases.verifySelectuserMenu(SalesforceUserMenu_Testcases.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySelectuserMenu -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceUserMenu_Testcases@39fcbef6]" started-at="2024-02-19T17:09:06 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:09:07 PST" duration-ms="907" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
      </class> <!-- com.salesforce.automation.SalesforceUserMenu_Testcases -->
    </test> <!-- TestUserMenu -->
    <test started-at="2024-02-19T17:09:07 PST" name="TestLeads&gt;" finished-at="2024-02-19T17:10:39 PST" duration-ms="91942">
      <class name="com.salesforce.automation.SalesforceLeads_Testcases">
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:07 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:09:09 PST" duration-ms="1707" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="Leadstablink()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:09 PST" name="Leadstablink" finished-at="2024-02-19T17:09:16 PST" duration-ms="6634" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Leadstablink -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:16 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:09:16 PST" duration-ms="119" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:16 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:09:18 PST" duration-ms="2236" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="defaultView()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:18 PST" name="defaultView" finished-at="2024-02-19T17:09:37 PST" duration-ms="18757" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- defaultView -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:37 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:09:53 PST" duration-ms="15744" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:53 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:09:54 PST" duration-ms="1545" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="leadsSelectView()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:09:54 PST" name="leadsSelectView" finished-at="2024-02-19T17:10:01 PST" duration-ms="6421" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- leadsSelectView -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:01 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:10:16 PST" duration-ms="15205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:16 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:10:17 PST" duration-ms="1601" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifyListitemTodaysLeads()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:17 PST" name="verifyListitemTodaysLeads" finished-at="2024-02-19T17:10:25 PST" duration-ms="7872" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//select[@id='fcf']"}
  (Session info: chrome=121.0.6167.184)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58549}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 30d580f5a78772de0811c13eff4d4e90
*** Element info: {Using=xpath, value=//select[@id='fcf']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//select[@id='fcf']"}
  (Session info: chrome=121.0.6167.184)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58549}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 30d580f5a78772de0811c13eff4d4e90
*** Element info: {Using=xpath, value=//select[@id='fcf']}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.salesforce.base.BaseClass.assertDefaultoption(BaseClass.java:504)
at com.salesforce.automation.SalesforceLeads_Testcases.verifyListitemTodaysLeads(SalesforceLeads_Testcases.java:129)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyListitemTodaysLeads -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:26 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:10:26 PST" duration-ms="125" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:26 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:10:27 PST" duration-ms="1707" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifyNewButtonOnLeads()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:27 PST" name="verifyNewButtonOnLeads" finished-at="2024-02-19T17:10:39 PST" duration-ms="11707" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNewButtonOnLeads -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceLeads_Testcases@6399551e]" started-at="2024-02-19T17:10:39 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:10:39 PST" duration-ms="149" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
      </class> <!-- com.salesforce.automation.SalesforceLeads_Testcases -->
    </test> <!-- TestLeads> -->
    <test started-at="2024-02-19T17:10:39 PST" name="TestRandomScenarios &gt;" finished-at="2024-02-19T17:11:20 PST" duration-ms="40838">
      <class name="com.salesforce.automation.SalesforceRandomScenarios_Testcases">
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:39 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:10:43 PST" duration-ms="3595" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="blockingAnEventCalender()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:43 PST" name="blockingAnEventCalender" finished-at="2024-02-19T17:10:48 PST" duration-ms="4746" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceRandomScenarios_Testcases.blockingAnEventCalender(SalesforceRandomScenarios_Testcases.java:298)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blockingAnEventCalender -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:48 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:10:49 PST" duration-ms="875" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:49 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:10:52 PST" duration-ms="3441" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifyAccountHolder()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:52 PST" name="verifyAccountHolder" finished-at="2024-02-19T17:10:53 PST" duration-ms="1302" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found [Login | Salesforce]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found [Login | Salesforce]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceRandomScenarios_Testcases.verifyAccountHolder(SalesforceRandomScenarios_Testcases.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAccountHolder -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:54 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:10:54 PST" duration-ms="809" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:54 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:10:58 PST" duration-ms="3327" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifylastnameEdit_Variousplace()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:10:58 PST" name="verifylastnameEdit_Variousplace" finished-at="2024-02-19T17:11:02 PST" duration-ms="4320" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceRandomScenarios_Testcases.verifylastnameEdit_Variousplace(SalesforceRandomScenarios_Testcases.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifylastnameEdit_Variousplace -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:02 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:11:03 PST" duration-ms="769" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:03 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:11:06 PST" duration-ms="3472" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifytabcustomization()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:06 PST" name="verifytabcustomization" finished-at="2024-02-19T17:11:11 PST" duration-ms="4450" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: //li[@id='AllTab_Tab']/a[@href]
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 120.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20231129155202, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 20.5.0, moz:processID: 49665, moz:profile: /var/folders/gx/qjsbcn6d6d3..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: e72bab5b-027e-4873-99d0-9bb185e3638d
*** Element info: {Using=xpath, value=//li[@id='AllTab_Tab']/a[@href]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: //li[@id='AllTab_Tab']/a[@href]
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 120.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20231129155202, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 20.5.0, moz:processID: 49665, moz:profile: /var/folders/gx/qjsbcn6d6d3..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: e72bab5b-027e-4873-99d0-9bb185e3638d
*** Element info: {Using=xpath, value=//li[@id='AllTab_Tab']/a[@href]}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.salesforce.automation.SalesforceRandomScenarios_Testcases.verifytabcustomization(SalesforceRandomScenarios_Testcases.java:205)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifytabcustomization -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:11 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:11:12 PST" duration-ms="806" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:12 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:11:15 PST" duration-ms="3176" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="weeklyrecurranceCalender()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:15 PST" name="weeklyrecurranceCalender" finished-at="2024-02-19T17:11:19 PST" duration-ms="4162" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Home Page ~ Salesforce - Developer Edition] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Home Page ~ Salesforce - Developer Edition] but found []
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceRandomScenarios_Testcases.weeklyrecurranceCalender(SalesforceRandomScenarios_Testcases.java:476)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- weeklyrecurranceCalender -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceRandomScenarios_Testcases@2d96543c]" started-at="2024-02-19T17:11:19 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:11:20 PST" duration-ms="947" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
      </class> <!-- com.salesforce.automation.SalesforceRandomScenarios_Testcases -->
    </test> <!-- TestRandomScenarios > -->
    <test started-at="2024-02-19T17:11:20 PST" name="TestContacts&gt;" finished-at="2024-02-19T17:12:35 PST" duration-ms="74641">
      <class name="com.salesforce.automation.SalesforceContact_Testcases">
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:20 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:11:22 PST" duration-ms="1679" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="checkCancelButtn_NewView()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:22 PST" name="checkCancelButtn_NewView" finished-at="2024-02-19T17:11:30 PST" duration-ms="8297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCancelButtn_NewView -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:30 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:11:30 PST" duration-ms="125" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:30 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:11:33 PST" duration-ms="2944" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="checkSaveNewButtn()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:33 PST" name="checkSaveNewButtn" finished-at="2024-02-19T17:11:42 PST" duration-ms="8471" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchWindowException">
            <message>
              <![CDATA[no such window
  (Session info: chrome=121.0.6167.184)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:59258}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: c65462b0f941009e14105a884ccdd222]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window
  (Session info: chrome=121.0.6167.184)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:59258}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: c65462b0f941009e14105a884ccdd222
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.window(RemoteWebDriver.java:903)
at com.salesforce.automation.SalesforceContact_Testcases.checkSaveNewButtn(SalesforceContact_Testcases.java:295)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchWindowException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkSaveNewButtn -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:42 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:11:43 PST" duration-ms="136" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:43 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:11:44 PST" duration-ms="1822" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="createNewContact()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:44 PST" name="createNewContact" finished-at="2024-02-19T17:11:53 PST" duration-ms="8528" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//table[@class='list']//tr[2]/th/a"}
  (Session info: chrome=121.0.6167.184)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:59306}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6a92a1359bf288a1bd16c9dbbee66941
*** Element info: {Using=xpath, value=//table[@class='list']//tr[2]/th/a}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//table[@class='list']//tr[2]/th/a"}
  (Session info: chrome=121.0.6167.184)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Nitins-MacBook-Pro.local', ip: '2601:646:8084:4d20:0:0:0:ec4a%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.4', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 121.0.6167.184, chrome: {chromedriverVersion: 121.0.6167.184 (057a8ae7deb..., userDataDir: /var/folders/gx/qjsbcn6d6d3...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:59306}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6a92a1359bf288a1bd16c9dbbee66941
*** Element info: {Using=xpath, value=//table[@class='list']//tr[2]/th/a}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.salesforce.automation.SalesforceContact_Testcases.createNewContact(SalesforceContact_Testcases.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewContact -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:53 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:11:53 PST" duration-ms="67" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:53 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:11:55 PST" duration-ms="2130" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="createNewViewContact()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:11:55 PST" name="createNewViewContact" finished-at="2024-02-19T17:12:04 PST" duration-ms="8526" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Accounts: Home ~ Salesforce - Developer Edition] but found [Contacts: Home ~ Salesforce - Developer Edition]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Accounts: Home ~ Salesforce - Developer Edition] but found [Contacts: Home ~ Salesforce - Developer Edition]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertTitle(BaseClass.java:256)
at com.salesforce.automation.SalesforceContact_Testcases.createNewViewContact(SalesforceContact_Testcases.java:105)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewViewContact -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:04 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:12:04 PST" duration-ms="113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:04 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:12:06 PST" duration-ms="1607" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="mycontactsView()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:06 PST" name="mycontactsView" finished-at="2024-02-19T17:12:14 PST" duration-ms="7546" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [View dropdwn selected] but found [w]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [View dropdwn selected] but found [w]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertDefaultoption(BaseClass.java:516)
at com.salesforce.automation.SalesforceContact_Testcases.mycontactsView(SalesforceContact_Testcases.java:137)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mycontactsView -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:14 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:12:17 PST" duration-ms="2500" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:17 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:12:18 PST" duration-ms="1762" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="recentContactsView()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:18 PST" name="recentContactsView" finished-at="2024-02-19T17:12:27 PST" duration-ms="8538" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- recentContactsView -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:27 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:12:27 PST" duration-ms="172" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
        <test-method is-config="true" signature="setUpBeforeMethod(java.lang.String)[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:27 PST" name="setUpBeforeMethod" finished-at="2024-02-19T17:12:29 PST" duration-ms="1893" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeMethod -->
        <test-method signature="verifyErrormessage()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:29 PST" name="verifyErrormessage" finished-at="2024-02-19T17:12:33 PST" duration-ms="4232" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://login.salesforce.com/] but found [https://dan000000cz3teag-dev-ed.develop.my.salesforce.com/setup/forcecomHomepage.apexp?setupid=ForceCom]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://login.salesforce.com/] but found [https://dan000000cz3teag-dev-ed.develop.my.salesforce.com/setup/forcecomHomepage.apexp?setupid=ForceCom]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.salesforce.base.BaseClass.assertCurrentURL(BaseClass.java:273)
at com.salesforce.automation.SalesforceContact_Testcases.verifyErrormessage(SalesforceContact_Testcases.java:181)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyErrormessage -->
        <test-method is-config="true" signature="tearDownAfterTestMethod()[pri:0, instance:com.salesforce.automation.SalesforceContact_Testcases@49d3c823]" started-at="2024-02-19T17:12:35 PST" name="tearDownAfterTestMethod" finished-at="2024-02-19T17:12:35 PST" duration-ms="145" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterTestMethod -->
      </class> <!-- com.salesforce.automation.SalesforceContact_Testcases -->
    </test> <!-- TestContacts> -->
  </suite> <!-- RegressionSuite -->
</testng-results>
